### PR TITLE
Update Hub + Landing pages to prep for L2 removal

### PR DESCRIPTION
**Pull request by wadepickett**
PR updates part 1 for #18011

[Internal Review - Hub](https://review.docs.microsoft.com/en-us/dotnet/?branch=pr-en-us-18024)
[Internal Review - Framework Landing Page](https://review.docs.microsoft.com/en-us/dotnet/framework/?branch=pr-en-us-18024)
[Internal Review - Standard Landing Page](https://review.docs.microsoft.com/en-us/dotnet/standard/?branch=pr-en-us-18024)
[Internal Review - What's New](https://review.docs.microsoft.com/en-us/dotnet/whats-new/?branch=pr-en-us-18024)
[Internal Review - Visual Basic Landing Page](https://review.docs.microsoft.com/en-us/dotnet/visual-basic/?branch=pr-en-us-18024)

Hub page:
- Add Download .NET highlight card to top left.
- Add a "Community" link plus a .NET Foundation link to the open source card

What's New page:
- Add a “Community” link to the “Get Involved” card.

.NET Framework landing page:
- Add link to .NET Download on "Install .NET Framework section"

.NET Standard landing Page:
- Add link to .NET Download as first link in Learn about .NET section

What's New page:
- Add Community and .NET Foundation links to "Get involved - contribute" section.

The L2 itself will be removed in a separate PR once these changes are merged.

### CODE REVIEW

File: docs/standard/index.yml [hunk 1]
```diff
@@ -18,6 +18,8 @@ landingContent:
     linkLists:
       - linkListType: overview
         links:
+          - text: Download the .NET SDK
```
Comment (BillWagner) [inline docs/standard/index.yml:21]: We've put the download link in a "download" linklisttype in other landing pages. We should be consistent.

File: docs/visual-basic/index.yml [hunk 2]
```diff
@@ -19,6 +19,8 @@ landingContent:
     linkLists:
       - linkListType: get-started
         links:
+          - text: Download the .NET Core SDK
```
Comment (BillWagner) [inline docs/visual-basic/index.yml:22]: Should download be in a download linked list type?

File: docs/whats-new/index.yml [hunk 3]
```diff
@@ -27,14 +27,18 @@ landingContent:
...
-  - title: "Get involved - contribute to .NET docs"
+  - title: "Get involved - contribute"
     linkLists:
       - linkListType: overview
         links:
           - text: ".NET docs repository"
             url: https://github.com/dotnet/docs/blob/master/README.md
           - text: Project structure and labels for issues and pull requests
             url: https://github.com/dotnet/docs/blob/master/styleguide/labels-projects.md
+          - text: "Community"
```
Comment (BillWagner) [inline docs/whats-new/index.yml:38]: It feels like these two should be in a separate list. They are distinct from contributing. It's bending it a bit, but the megaphone (whatsnew) seems to fit correctly, and it's what we've used on other pages. Thoughts?

**Review by BillWagner — APPROVED** [review]
This looks great @wadepickett 

I looked at the other landing pages for consistency with what we've done on other landing pages and made a couple suggestions. Once you decide what to do, let's :shipit:

File: docs/whats-new/index.yml [hunk 4]
```diff
@@ -27,14 +27,18 @@ landingContent:
...
 
-  - title: "Get involved - contribute to .NET docs"
+  - title: "Get involved - contribute"
     linkLists:
       - linkListType: overview
         links:
           - text: ".NET docs repository"
             url: https://github.com/dotnet/docs/blob/master/README.md
           - text: Project structure and labels for issues and pull requests
             url: https://github.com/dotnet/docs/blob/master/styleguide/labels-projects.md
```
Comment (tdykstra) [inline docs/whats-new/index.yml:37]:
```suggestion
            url: https://docs.microsoft.com/en-us/contribute/dotnet/labels-projects
```

File: docs/framework/index.yml [hunk 5]
```diff
@@ -44,6 +44,8 @@ landingContent:
         links:
           - text: Installation guide 
             url: install/index.md
+          - text: Download the .NET SDK
```
Comment (gewarren) [inline docs/framework/index.yml:47]: This should go under a "download" linkListType too.

File: docs/index.yml [hunk 6]
```diff
@@ -99,7 +109,7 @@ conceptualContent:
         - url: core/sdk.md
           itemType: download
           text: Download the .NET Core SDK
-        - url: core/tools/index.md
+        - url: https://dotnet.microsoft.com/download
```
Comment (gewarren) [inline docs/index.yml:112]: Was this supposed to go on line 109 @wadepickett ?

Comment (wadepickett) [reply docs/index.yml:112]: Just a note so I don't think I forgot to reply on this one. The fix and discussion for this moved to #18159.  Thanks @gewarren!

**Pull request merged**